README.md

```diff
@@ -25,13 +25,39 @@ Or install it yourself as:
 ```ruby
 require 'zeus/api_client'
 zeus_client = Zeus::APIClient.new({
-    :access_token => "your_token_here",
+    :token => "your_token_here",
     :endpoint => "zeus_endpoint_here"
 })
 ```
 
 List All Metrics
 
+If you use user token + `org_name/bucket_name`:
+```ruby
+result = zeus_client.bucket("your_org_name/your_bucket_name").list_metrics(
+  # optional parameters
+  {
+    regex: "metric.name*" # regex for filtering metrics names
+    from_date: "2016-06-12", # YYYY-MM-DD
+    to_date: "2017-12-20", # YYYY-MM-DD
+    aggregator_function: "sum", # Aggregator function to be applied to the metric values.
+                                # Available options. count, min, max, sum, mean, mode, median
+    group_interval: "100m",     # Interval for grouping, when applying aggregations via the aggregator_function parameter.
+                                # The aggregator_function parameter must be specific for this parameter to work.
+                                # Examples: 1000s, 100m, 10h, 1d , 1w. Use 's' for seconds, 'm' for minutes, 'h' for hours,
+                                # 'd' for days and 'w' for weeks.
+    filter_condition: "column1 > 0", # Filters to be applied to metric values.
+                                     # Examples: "column1 > 0", "column1 > 50 AND column2 = 10".
+    offset: 100,
+    limit: 10
+  }
+)
+p result.code      # 200
```

Comment:
I think README become too long, so maybe we can delete these status code example.